core/client/src/main/java/alluxio/client/file/UnderFileSystemFileOutStream.java

```diff
@@ -73,7 +73,7 @@ protected Factory() {} // prevent external instantiation.
      * @param ufsFileId the file ID of the ufs fild to write to
      * @return a new {@link UnderFileSystemFileOutStream}
      */
-    public UnderFileSystemFileOutStream create(InetSocketAddress address, long ufsFileId) {
+    public OutputStream create(InetSocketAddress address, long ufsFileId) {
```

Comment:
Since `UnderFileSystemFileOutStream` doesn't expose any other methods aside from those in `OutputStream`, this return type should be equivalent. It also makes it symmetric with `DelegatedUnderStoreStreamFactory`.